src_plug/alltests/main.cpp

```diff
@@ -9,9 +9,18 @@ void OnPluginLoad(EPluginLoadingResult* pResult_)
 {
     trap_Com_Printf("[alltests]: Hello, %s world!\n", "formatted");
     trap_Com_PrintError("[alltests]: Com_PrintError(): Test Com_PrintError %d\n", 1);
-    trap_Cbuf_AddText("developer 1;wait 50;map mp_vacant\n");
+    //trap_Cbuf_AddText("developer 2\n");
     trap_Com_DPrintf("[alltests]: Com_DPrintf(): Com_DPrintf %s !\n", "test");
     trap_Com_PrintWarning("[alltests]: Com_PrintWarning(): Com_PrintWarning %s !\n", "test");
+    trap_SV_SetConfigString(1000, "defaultweapon");// :D
+    char buffer[8192];
```

Comment:
C++, so
`char buffer[8192] = {'\0'};` 
Means "Set first element to null char and nullify rest"